en/bundle.ftl

```diff
@@ -11,20 +11,22 @@ vpn-relay-go-vpn-cta = Download { -brand-name-mozilla-vpn }
 
 ## VPN and Relay Bundle What's New Announcement
 
-whatsnew-feature-bundle-header = Introducing: { -brand-name-relay } + { -brand-name-vpn } subscription plan
-whatsnew-feature-bundle-snippet = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn }…
+# Variables:
+#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+whatsnew-feature-bundle-header = Add { -brand-name-mozilla-vpn } for { $savings } off!
+whatsnew-feature-bundle-snippet = Get a year of { -brand-name-relay-premium } Phone Masking & Email Masking plus { -brand-name-mozilla-vpn }…
 # Variables:
 #   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
 #   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
-whatsnew-feature-bundle-body = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month. Save { $savings }!
+whatsnew-feature-bundle-body = Get a year of { -brand-name-relay-premium } Phone Masking & Email Masking plus { -brand-name-mozilla-vpn } for { $monthly_price }/month.
 whatsnew-feature-bundle-upgrade-cta = Upgrade now
 
-## VPN and Relay Bundle Banner
+## VPN and Relay Landing Page Bundle Banner
 
 bundle-banner-header = { -brand-name-firefox-relay } with <vpn-logo>{ -brand-name-mozilla-vpn }</vpn-logo>
 bundle-banner-subheader = Security, reliability and speed — on every device, anywhere you go.
 bundle-banner-body = Surf, stream, game, and get work done while maintaining your privacy online. Whether you’re traveling, using public Wi-Fi, or simply looking for more online security, we will always put your privacy first.
-bundle-banner-1-year-plan = 1 year plan: <b>{ -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn }</b>
+bundle-banner-1-year-plan = 1 year plan: <b>{ -brand-name-firefox-relay } Email & Phone Masking + { -brand-name-vpn }</b>
```

Comment:
Updates the bundle banner on landing page. 
<img width="537" alt="image" src="https://user-images.githubusercontent.com/13066134/197018207-2090e611-259b-4273-9ba5-e6fed3826a32.png">